### PR TITLE
feat(espresso): add deployments for upcoming demo

### PR DESCRIPTION
**Pull request by james-a-morris**
Adds deployment definitions for the upcoming Espresso demonstration. Two custom rollups (id 10,001 & 10,002) have been created. 

Note: since no canonical bridge exists, the corresponding chain-adapter's `relayMessage` is a no-op & the spoke pools' `_requireAdminSender` is also a no-op. These are _very bad_ candidates for testing the dataworker.

Note: based on the sequencer, these two proxy addresses have the same address. This is intended

### CODE REVIEW

File: hardhat.config.ts [hunk 1]
```diff
@@ -125,7 +125,7 @@ const config: HardhatUserConfig = {
       verifyURL: "https://zksync2-mainnet-explorer.zksync.io/contract_verification",
     },
     optimism: {
-      url: getNodeUrl("optimism", true, 10),
+      url: getNodeUrl("optimism-mainnet", true, 10),
```
Comment (nicholaspai) [inline hardhat.config.ts:128]: Is this right? How has this worked before?

**Review by nicholaspai — APPROVED** [review]
One q

File: hardhat.config.ts (hunk 1 above)
Comment (james-a-morris) [reply hardhat.config.ts:128]: Yeah - the first variable is the subdomain. On infura it's `optimism-mainnet` not `optimism`

**Comment by james-a-morris** [comment]
Can close because the demo is complete.

**Pull request closed without merging**